frontend/javascripts/oxalis/model/actions/volumetracing_actions.js

```diff
@@ -23,7 +23,7 @@ type AddToLayerAction = { type: "ADD_TO_LAYER", position: Vector3 };
 type FloodFillAction = { type: "FLOOD_FILL", position: Vector3, planeId: OrthoView };
 type FinishEditingAction = { type: "FINISH_EDITING" };
 type SetActiveCellAction = { type: "SET_ACTIVE_CELL", cellId: number };
-type SetToolAction = { type: "SET_TOOL", tool: VolumeTool };
+type SetToolAction = { type: "SET_TOOL", tool: AnnotationTool };
```

Comment:
These two actions should be moved to the annotation actions (as you did for the respective reducers)

Reply:
good point!